app/views/users/show_follow.html.erb

```diff
@@ -36,7 +36,7 @@
                         data-img="<%= user.profile_image %>" 
                         data-content='<div class="card">
                                         <div class="media m-2">
-                                          <img src="<%= user.profile_image %>" class="mr-3" alt="">
+                                          <img src="<%= user.profile_image %>" height=80px width=80px class="mr-3" alt="">
```

Comment:
This looks great, quick question...should the pixels be included in double quotes? Similar to what is happening [here](https://www.w3schools.com/tags/att_img_width.asp)? cc @17sushmita

Reply:
Thanks for review. According to [this](https://html.spec.whatwg.org/#a-quick-introduction-to-html):

> Attributes are placed inside the start tag, and consist of a name and a value, separated by an "=" character. **The attribute value can remain unquoted if it doesn't contain ASCII whitespace or any of " ' ` = < or >. Otherwise, it has to be quoted using either single or double quotes.** The value, along with the "=" character, can be omitted altogether if the value is the empty string.